inlong-sort/sort-connectors/hbase/src/main/java/org/apache/inlong/sort/hbase/sink/HBaseSinkFunction.java

```diff
@@ -128,18 +129,18 @@ public void open(Configuration parameters) throws Exception {
             this.runtimeContext = getRuntimeContext();
             sinkMetricData = new SinkMetricData(runtimeContext.getMetricGroup());
```

Comment:
here if sinkMetricData would throw npe if inlongMetrics is null. So the initialization should be after the check of inlongMetrics.
if (inLongMetric != null && !inLongMetric.isEmpty()) { new sinkMetricData  ... }